resteasy-jaxrs/src/main/java/org/jboss/resteasy/microprofile/config/ResteasyConfig.java

```diff
@@ -1,18 +1,24 @@
 package org.jboss.resteasy.microprofile.config;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import java.security.AccessController;
 import java.security.PrivilegedActionException;
 import java.security.PrivilegedExceptionAction;
+import java.util.Optional;
 
 import javax.servlet.ServletContext;
 
-import org.eclipse.microprofile.config.Config;
+import org.jboss.resteasy.resteasy_jaxrs.i18n.LogMessages;
 import org.jboss.resteasy.resteasy_jaxrs.i18n.Messages;
 import org.jboss.resteasy.spi.ResteasyProviderFactory;
 
 public class ResteasyConfig
 {
-   private Config config;
+   private static boolean MP_CONFIG_PRESENT;
+   private static final Method GET_OPTIONAL_VALUE;
+   private static final Method GET_PROPERTY_NAMES;
+   private static Object config;
```

Comment:
I don't think we want this to be static. There is a change we want a different one per deployment.

Reply:
Well, RESTEasy's servlet ConfigSources should find the appropriate ServletContext, etc., but it couldn't hurt. Ok, thanks.